nautobot/docs/user-guide/administration/upgrading/from-v1/ipam-upgrade-guide.md

```diff
@@ -0,0 +1,455 @@
+# Nautobot 2.0 IPAM Migration Guide
+
+## Goal
+
+This document aims to answer the question “What do I need to do to my data (or what changes with my data) when migrating to 2.0, specifically IPAM?”
+
+## What Changed
+
+This section details the high-level changes as it relates to the data modeling for IPAM objects. Some of the sections are repeating content found in the release notes with detail added for improved readability. Additionally, not all changes described will be distinctly related to IPAM, but if there were changes to relationships to or from IPAM data models, the impact of those changes will be described here.
+
+### Namespaces were introduced
+
+The new `Namespace` model expands on the functionality previously provided by `VRF.enforce_unique` and the `ENFORCE_GLOBAL_UNIQUE` setting, both of which have now been removed. Within a Namespace, all VRFs, Prefixes, and IP addresses must be unique. This enables greater flexibility in managing discrete duplicate `VRF`, `Prefix` or `IPAddress` objects, asserting that each set of duplicates will be in a distinct `Namespace`.
```

Comment:
> This enables greater flexibility in managing discrete duplicate `VRF`, `Prefix` or `IPAddress` objects, asserting that each set of duplicates will be in a distinct `Namespace`.

I don't have a good suggestion here but I don't feel that this explains how or why namespaces are making managing duplicates more flexible. Perhaps state the equivalence of namespaces to distinct layer 3 routing tables and add that we've correct previous modeling that required duplicate IPs/Prefixes/VRFs?

Reply:
Namespaces are functionally equivalent to distinct layer 3 routing tables. The introduction of Namespaces corrects previous modeling that required duplicate `IPAddress`, `Prefix`, and `VRF` objects. The ultimate goal of Namespaces is to enable greater flexibility in managing discrete duplicate `VRF`, `Prefix` or `IPAddress` objects, asserting that any set of duplicates objects will no longer be necessary by way of the core data model itself and instead becomes an intuitive way to explicitly manage duplicate IPAM objects within distinct `Namespaces`.

🤷‍♂️